ansible/playbooks/roles/explorer_be/files/leaders.json

```diff
@@ -0,0 +1,9 @@
+$ANSIBLE_VAULT;1.1;AES256
```

Comment:
ditto: what are steps to update this file when explorer nodes change?

Reply:
i think we need another Ansible script to update leaders, before it maybe we can change it manually